modules/rules_proto_grpc_python/5.0.0-alpha2/MODULE.bazel

```diff
@@ -0,0 +1,41 @@
+module(
+    name = "rules_proto_grpc_python",
+    version = "5.0.0-alpha2",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "grpc", version = "1.48.1.bcr.2")
+bazel_dep(name = "protobuf", version = "21.7")
+bazel_dep(name = "rules_proto_grpc", version = "5.0.0-alpha2")
+bazel_dep(name = "rules_python", version = "0.27.1")
+
+PYTHON_VERSIONS = [
+    "3.8",
+    "3.9",
+    "3.10",
+    "3.11",
+    # "3.12",  # Not yet working
+]
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
```

Comment:
I think that this should be a `dev_dependency`. If this module is used by another module, it probably shouldn't supply Python toolchains.

Reply:
Will this affect how the pip install below it works, as that isn't a dev dep? If not, sure I agree that can become dev dep

Reply:
I tested adding `dev_dependency` to that line and it breaks the pip extension in the following lines, even when the root module loads `rules_python` and declares a single toolchain. So unfortunately, with the current structure of the Python rules, it doesn't appear possible

Reply:
Thanks for trying, this ought to be fine then.